tools/spec2vec/spec2vec_training.xml

```diff
@@ -0,0 +1,254 @@
+<tool id="spec2vec_training" name="Spec2Vec Model Training" version="@TOOL_VERSION@-@TOOL_DEV_VERSION@+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>Train a Spec2Vec model for mass spectra similarity scoring</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator"/>
+
+    <requirements>
+        <container type="docker">recetox/spec2vec:@COMMIT_SHA@</container>
```

Comment:
I have created a conda package here: https://github.com/bioconda/bioconda-recipes/pull/38439

This will come automatically with a container. Would it be possible to use the package definition here?

Reply:
We actually discussed this with @florian-huber - the dockerfile allows us to test development builds on galaxy so to say, since we can always build and upload a docker container and then for the correct versions we plan to use the conda package - so this comes in handy!

Reply:
You are very welcome! Thanks for all your effort in those cool tools!